pkgs/servers/interlock/default.nix

```diff
@@ -34,4 +33,18 @@ buildGoPackage rec {
       -e 's|/usr/bin/sudo|/run/wrappers/bin/sudo|' \
       -e 's|/sbin/cryptsetup|${cryptsetup}/bin/cryptsetup|'
   '';
+
+  postInstall = ''
+    mkdir -p $out/share
+    cp -R $src/static $out/share
+  '';
+
+  doCheck = false;
```

Comment:
a comment why we skip the checks would be nice

Reply:
I've added a comment. I didn't manage to figure out why the test fails tho. Could be something related to filesystem or inability to generate OpenPGP keys for some reason, even though it's done via a library which shouldn't depend on anything.